google-chart.html

```diff
@@ -346,7 +240,7 @@
     // The chart type was changed.
     // We need to create a new chart and redraw.
     _typeChanged: function() {
-      this._chart = createChart(this.type, this.$.chartdiv)
+      this._chart = this.$.loader.create(this.type, this.$.chartdiv)
```

Comment:
if you wrap the loader in a dom-if, this would need to be rewritten. Maybe not worth it.